code/_onclick/hud/hud.dm

```diff
@@ -187,3 +187,5 @@
 	else
 		to_chat(usr, "<span class ='warning'>This mob type does not use a HUD.</span>")
 
+/image
+	plane = FLOAT_PLANE // I have no fucking clue why this isn't FLOAT_PLANE by default in BYOND.
```

Comment:
Yes, I've tested without this. Without this, all the overlays would not show up, because they were rendering behind the object they were overlaying.